tooling/release_download_test.sh

```diff
@@ -8,66 +8,191 @@
 #   1 - Something fundamentally wrong before we could check anything
 #   2 - GPG signature verification failed
 #   3 - SHA checksum failed
-#   4 - aarch64 detected GCC/GLIBC version not as expected
+#   4 - detected GCC/GLIBC version not as expected
 #   5 - CylconeDX validation checks failed
 #   6 - SBOM contents did not meet expectations
 # Note that if there are multiple failures the highest will be the exit code
 # If there is a non-zero exit code check the output for "ERROR:"
 # 
 # For future enhancement ideas, see https://github.com/adoptium/temurin-build/issues/3506#issuecomment-1783237963
 #
+
+set -euo pipefail
+
 STARTDIR="$PWD"
-TAG=${1:-$TAG}
-[ -z "$TAG" ] && echo "Usage: $0 TAG" && exit 1
-[ "$(uname -m)" != "aarch64" ] && echo "This script is hard coded to be run on Linux/aarch64 - aborting" && exit 1
+WORKSPACE=${WORKSPACE:-"$PWD"}
+VERBOSE=false
+KEEP_STAGING=false
+SKIP_DOWNLOADING=false
+
+NORMAL=""
+BOLD=""
+RED=""
+YELLOW=""
+
+# check if stdout is a terminal...
```

Comment:
Useful function, extract to a util script for all scripts perhaps?

Reply:
this has been extracted into a common_logging.sh script